install_config/provisioners.adoc

```diff
@@ -153,7 +155,8 @@ The following command sets the directory in the EFS volume to
 be mountable and writeable by the provisioner pod.
 
 ----
-$ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/playbooks/openshift-provisioners/config.yml \
+$ ansible-playbook -v -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/openshift-provisioners.yml \
```

Comment:
@gpei Thank you for reviewing this. I have made changes in this one for 3.6 and 3.7. After I merge this, I will create another PR for 3.9 specific changes.